website/docs/r/azurerm_server_vulnerability_assessment.html.markdown

```diff
@@ -0,0 +1,111 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_server_vulnerability_assessment"
+description: |-
+  Adds Azure Vulnerability Assessment (Qualys) to a VM or Azure ARC server.
```

Comment:
It could be updated to "Manages an Azure Vulnerability Assessment (Qualys) to a VM or Azure ARC server" to align the style with other existing resources. Does it make sense?